README.md

```diff
@@ -13,7 +13,7 @@ The `@lukso/eip191-signer.js` package is used to sign an EIP191 Execute Relay Ca
 
 This library will add the `\x19Execute Relay Call:\n` prefix to a message and sign it.
```

Comment:
We have to update this description as now EIP 191 does not add the prefix mentioned anymore.

See EIP191 specs + Key Manager specs

https://eips.ethereum.org/EIPS/eip-191

https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-6-KeyManager.md#executerelaycall